test/factories.ts

```diff
@@ -120,6 +121,26 @@ export const optionSelectFactory = new Factory()
     type: 'optionSelect',
   });
 
+export const hiddenFactory = new Factory()
```

Comment:
How about creating a `typeFactory` function that takes a type argument, since this most of these have no other attributes? Not necessary but it would clean this section up considerably

Reply:
I hate that this never occurred to me. That cleanup was INCREDIBLE.